starter-code/javascript/main.js

```diff
@@ -7,54 +7,85 @@ var secDec      = document.getElementById('secDec');
 var secUni      = document.getElementById('secUni');
 var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
+var splits = document.getElementById("splits");
 
 
 function printTime() {
-
-}
+    setInterval(() => {
+        printMinutes();
+        printSeconds();
+        printMilliseconds();
+    }, 1000);
+};
```

Comment:
You should set a lesser interval to update the dom, otherwise you get a little delay when you click on the reset button.
 
   ```
setInterval(() => {
        printMinutes();
        printSeconds();
        printMilliseconds();
    },100);
};
```